.gitignore

```diff
@@ -126,3 +126,7 @@ ENV/
 
 # ts lib files
 !srcmedia/ts/lib/
+
+notebooks
+data
+*dwellings*
```

Comment:
these seem like things that belong in a personal gitignore rather than the project-wide gitignore